mautic/mautic
# Add UI update deprecation notice

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:

If Composer updates are disabled:

<img width="1718" alt="image" src="https://user-images.githubusercontent.com/17739158/156000635-28c341fa-2a19-498e-bff4-e8d4edb0c363.png">

If Composer updates are enabled:

<img width="1717" alt="image" src="https://user-images.githubusercontent.com/17739158/156000709-b8ab0343-622e-4e05-ad09-d4f7f41bf923.png">



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. In `app/config/local.php` temporarily add the following lines (or replace if they're already there): 
```PHP
'system_update_url' => 'https://api.github.com/repos/dennisameling/mautic/releases',
'update_stability' => 'alpha',
```
5. Go to https://mautic.ddev.site/index_dev.php/s/update. You should see the messages as shown in the screenshots




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10907"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
